test/exponential.jl

```diff
@@ -4,10 +4,9 @@ using IntervalMatrices: TaylorOverapproximation,
                         Horner
 
 @testset "Interval matrix exponential" begin
+    @test quadratic_expansion(-3 .. 3, 1.0, 2.0) == Interval(-0.125, 21)
 
-    @test quadratic_expansion(-3..3, 1.0, 2.0) == Interval(-0.125, 21)
-
-    M = IntervalMatrix([-1.1..0.9 -4.1.. -3.9; 3.9..4.1 -1.1..0.9])
+    M = IntervalMatrix([-1.1 .. 0.9 -4.1 .. -3.9; 3.9 .. 4.1 -1.1 .. 0.9])
```

Comment:
I had to manually add the spaces here. The formatter wants to write `-4.1..-3.9`, which is a syntax error.